Mathlib/Tactic/Congr.lean

```diff
@@ -0,0 +1,112 @@
+import Lean.Meta
+import Mathlib.Mathport.Syntax
```

Comment:
You should do it the other way around: move the syntax for `congr` into this file and have `Mathlib.Mathport.Syntax` import `Mathlib.Tactic.Congr`

Reply:
Will do this.